int-tests/server_controller.py

```diff
@@ -69,7 +69,9 @@ def __init__(self, script, ready_str, *args, **kwargs):
     self._name = kwargs.get('name', 'unnamed')
     self._verbose = kwargs.get('verbose', False)
     if len(args) > 1:
-      argv = functools.reduce(lambda a, b: ("", str(a).strip() + " " + str(b).strip()), args)[1]
+      argv = ' '
+      for each in [str(i).strip() for i in args if len(str(i)) > 0]:
+        argv += ' ' + each
```

Comment:
I always cry a little bit when a lambda dies, but this list comprehension is more readable.